doc/setup/install.rst

```diff
@@ -137,6 +137,11 @@ When the configuration file is edited, Odoo must be restarted using
     $ sudo service odoo restart
     Restarting odoo: ok
 
+.. warning:: if the configuration file :file:`/etc/odoo/openerp-server.conf`
```

Comment:
We don't want to break scripts of people already having a custom configuration in /etc/odoo/openerp-server.conf. With this fallback, odoo will continue using the configuration in this file, unless the user deletes it.